packages/common/src/ui/components/navigation/NavLink/NavLink.test.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC, useEffect } from 'react';
```

Comment:
stories and tests for `NavLink` and `Breadcrumbs` which I also put into a `components/navigation` folder